### ISSUE TITLE
Killer visualisations for the paper

### ISSUE THREAD

**Issue by ben18785** [pints-team/comp-bio-paper#2]
Dave suggested that we have a panel of figures early in the paper which illustrates the difficulty of the inverse problem for time series modelling. We were thinking of, perhaps, having:

- Ross' plot of the jagged nature of a log-likelihood surface when too long a time step is used.
- Perhaps a voltage/current trace from electrophysiology, illustrating the difficult nature of the solving ODE/PDEs. I am very naive here and so any other suggestions would be really welcome.
- Perhaps another two.

Ideas? 

Also, if someone knows where I can get Ross' figure from/reproduce it, let me know.

**Comment by sanmitraghosh** [comment]
Take a logistic regression model (not the logistic ODE, @ben18785  can do that in R). Plot the likelihood/posterior surface. Then do the same for Fitzhugh-Nagumo model, or a simple van-der-pol oscillator. These two figures side by side is all that's needed to communicate the difficulties. Other visualisations of difficulties are irrelevant. Ross's tolerances plot is good but too domain specific, I can give many counter examples.

**Comment by ben18785** [comment]
@MichaelClerx Good point re: Ross' model. I agree that it doesn't really exemplify the need for PINTS and so may be better in a separate paper.

@sanmitraghosh thanks. Will take a look at this model. Also do you have any good references for the difficulty of inferring ODEs/PDEs?

**Comment by sanmitraghosh** [comment]
@ben18785 [this paper Fig 5](https://arxiv.org/pdf/1404.5053.pdf). Best example of how general good statistical intuitions fail for complex ODE systems, because of the likelihood/posterior surface geometries.

**Comment by ben18785** [comment]
Thanks Sanmitra. I am going to give the Goodwin model from Calderhead and Girolami 2009 a go. It may be a good case study for use of populationMCMC. 

Having read some of the Calderhead papers, I do think getting the second order sensitivity samplers - whilst not useful, perhaps for electrochemistry and cardiac stuff - will in general be in demand from systems bio models.

**Comment by sanmitraghosh** [comment]
Ya I know, but I am so much bogged down with other stuff that I can't finish smMALA which would work quite well for these sort of systems.

**Comment by ben18785** [comment]
Key paper: https://royalsocietypublishing.org/doi/pdf/10.1098/rsfs.2011.0051
